JavaScript/Task_Day2/MiddleCharacter.js

```diff
@@ -0,0 +1,9 @@
+//https://www.codewars.com/kata/56747fd5cb988479af000028/train/javascript
+
+//Get the Middles Character
+
+function getMiddle(str) {
+  var len = str.length;
+  if (len % 2 == 0) return str.substring(len / 2 - 1, len / 2 + 1);
```

Comment:
Always try to use === instead ==.
since triple equal to will even check the type of the operands, it is a good practice to use === instead of ==